optimism/src/keccak/trace.rs

```diff
@@ -56,7 +56,26 @@ impl<F: Field> CircuitTrait<ZKVM_KECCAK_COLS, Steps, F, KeccakEnv<F>> for Keccak
         });
     }
 
-    fn pad(&mut self, step: Steps) -> bool {
+    fn pad_with_row(&mut self, step: Steps, row: &[F; ZKVM_KECCAK_COLS]) -> bool {
+        let mut step = step;
+        if let Round(_) = step {
+            step = Round(0);
```

Comment:
I don't understand this condition. Can you explain, please?

Reply:
Added a comment, will push in a few min